packages/server/tests/integration/test.env

```diff
@@ -1,2 +1,3 @@
 E2E_SERVER_PORT=7546
 CHAIN_ID=0x12a
+TEST=true
```

Comment:
qq: i might have missed it but what's the reason for this config again?

Reply:
It’s to use LRU cache instead of expecting a redis server to be up in the unit tests